### DISCUSSION TITLE
Cannot change the checkboxes on the website.

### DISCUSSION THREAD

**Discussion by snippet2021** [yasserfarouk/scml#107] [General]
I logged in to change my entry track, but all the check boxes are set to DISABLE and I am unable to change my selection.

**Comment by yasserfarouk** [comment]
This will be fixed in a hour or so. Just in case please let me know your agent’s id and track(s).

**Reply by yasserfarouk** [reply]
The checkboxes are now temporarily enabled (until the report submission deadline). 
Please let me know your agent ID and track(s) just in case.
thanks

**Comment by snippet2021** [comment]
Team id:137 ->std only

Now we can't do anything like this.

 ![image](https://user-images.githubusercontent.com/92664206/174058846-0a7c01eb-5823-49fe-97ba-3abc2305d033.png)

**Reply by yasserfarouk** [reply]
Resolved. Sorry for the inconvenience. You should be able to upload your report now. Please remember to press "update" after you choose the report file.

**Reply by snippet2021** [reply]
Now,we have another trouble.
When we check col track and push update button, std track was checked
In the other hands,when we check std track and push update button, col track was checked

When we check no track and click update button,col and std was checked.

**Reply by yasserfarouk** [reply]
It is working now. Sorry for the inconvenience!!

**Reply by snippet2021** [reply]
Thank you, I have confirmed the correction.
